base/ApplicationSupport.F90

```diff
@@ -39,8 +39,8 @@ subroutine MAPL_Initialize(unusable,comm,logging_config,rc)
       call initialize_pflogger(comm=comm_world,logging_config=logging_configuration_file,rc=status)
       _VERIFY(status)
 #endif
-      call start_global_profiler(comm=comm_world,rc=status)
-      _VERIFY(status)
+      call initialize_profiler(comm=comm_world)
```

Comment:
I think this will need an rc, because the global profiler might return an invalid status.

Reply:
there is no rc in initialize and finalize. I added rc to start and stop